File: API/Core/Entities/Page.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace API.Core.Entities
{
    public class Page : BaseEntity
    {
        public string? Title { get; set; }
        public string? ImageUrl { get; set; }
        public DateTime CreatedTime { get; set; } = DateTime.Now;
        public string? Content { get; set; }
        public bool IsPublic { get; set; }
        public string? AppUserId { get; set; }
        public AppUser? AppUser { get; set; }
        public int TagId { get; set; }
        public Tag? Tag { get; set; }
    }
}

File: client/src/app/shared/models/page.ts
export interface IPage {
  id: number;
  title: string;
  imageUrl: any;
  createdTime: Date;
  content: string;
  isPublic: boolean;
  appUserId: string;
  tagId: number;
}

export class Page {
  id: number = 0;
  title: string = '';
  imageUrl: any;
  createdTime: Date = new Date();
  content: string = '';
  isPublic: boolean = false;
  appUserId: string = '';
  tagId: number = 0;
}
